src/containers/App.js

```diff
@@ -0,0 +1,98 @@
+import React, {Component} from 'react';
+
+import './App.css';
+import Inventory from '../components/Inventory/Inventory';
+import Cart from '../components/Cart/Cart';
+
+class App extends Component {
+    state = {
+        inventory: [
+            {id: "one", name: "fish", price: 50, quantity: 5},
+            {id: "two", name: "rice", price: 50, quantity: 4},
+            {id: "three", name: "beans", price: 70, quantity: 3}
+        ],
+        cart: [],
+        showCart: false,
+    };
+
+    addToCart = (id) => {
+        const inventoryIndex = this.state.inventory.findIndex(p => {
```

Comment:
you can refactor this operation to be a `filter` and you return the output of the filter